src/init/init.c

```diff
@@ -54,6 +54,10 @@ int master_core_init(struct sof *sof)
 	trace_point(TRACE_BOOT_SYS_POWER);
 	pm_runtime_init();
 
+	/* Turn off memory for all unused cores*/
+	for (i = 1; i < PLATFORM_CORE_COUNT; i++)
```

Comment:
Don't assume, that master core is 0. Iterate over all the cores and do it for `i != PLATFORM_MASTER_CORE`.

Reply:
@tlauda actually here it doesnt matter which one is master so the comparison is not needed